homeassistant/components/prosegur/__init__.py

```diff
@@ -0,0 +1,52 @@
+"""The Prosegur Alarm integration."""
+import asyncio
+
+from pyprosegur.auth import Auth
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import aiohttp_client
+
+from .const import DOMAIN
+
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
+
+PLATFORMS = ["alarm_control_panel"]
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the Prosegur Alarm component."""
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up Prosegur Alarm from a config entry."""
+    session = aiohttp_client.async_get_clientsession(hass)
+    hass.data[DOMAIN] = {}
+    hass.data[DOMAIN][entry.entry_id] = Auth(
+        session, entry.data["username"], entry.data["password"], entry.data["country"]
```

Comment:
```suggestion
        session, entry.data[CONF_USERNAME], entry.data[CONF_PASSWORD], entry.data[CONF_COUNTRY]
```

Best would be to use constants in your integration, you can import `CONF_PASSWORD, CONF_USERNAME` from `homeassistant.const` and `CONF_COUNTRY` from your own const. (or use a const like CONF_REGION from `homeassistant.const`.